src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -126,6 +126,15 @@
   <data name="ApprovedCipherModeMessage" xml:space="preserve">
     <value>It uses an unsafe Cipher Mode {0}</value>
   </data>
+  <data name="DoNotDisableCertificateValidation" xml:space="preserve">
+    <value>Do Not Disable Certification Validation</value>
+  </data>
+  <data name="DoNotDisableCertificateValidationDescription" xml:space="preserve">
+    <value>The certificate establish a trust between the recipient of a signed message and the signer of the message. Each recipient of a signed message need to decide if the certificate is trustworthy. If the ServerCertificateValidationCallback always return 'true', it will pass the validation and trust all certificates.</value>
```

Comment:
How about something like: "A certificate can help authenticate the identity of the server.  Clients should validate the server certificate to ensure requests are sent to the intended server.  If the ServerCertificateValidationCallback always returns 'true', any certificate will pass validation."